qpc/tests/cred/test_openshift_cred_add.py

```diff
@@ -91,10 +91,7 @@ def test_add_no_token(
             CLI().main()
         out, err = capsys.readouterr()
         assert out == ""
-        assert (
-            "one of the arguments --password --sshkeyfile"
-            " --sshkey --token is required" in err
-        )
+        assert "one of the arguments --password" " --sshkey --token is required" in err
```

Comment:
```suggestion
        assert "one of the arguments --password --sshkey --token is required" in err
```

I think black tends to no join strings where they used to be on multiple lines. Not that it really matters.

Reply:
Good catch. This happened as a result of `ruff` autoformatting the file, and I missed it. I'll fix this manually.